api/python/requirements/install.txt

```diff
@@ -0,0 +1,5 @@
+salt == 3002.2
+PyYAML
+Jinja2
+packaging >= 16.8
+requests >= 2.14.2
```

Comment:
Salt 3002 comes with 2.12.5-4 
https://repo.saltproject.io/py3/redhat/7/x86_64/3002/

We should maintain that as our baseline:
```requests >= 2.12.5```

Reply:
epel provides 2.14.2 and it is minimal one we can consider. In case salt repo is missed by some reason or salt repo drops the usage of requests we will fail. So current version takes the best from both options.